sklearn/cluster/optics_.py

```diff
@@ -562,10 +573,17 @@ def _extract_optics(ordering, reachability, maxima_ratio=.75,
         labels[index] = clustid
         is_core[index] = 1
         clustid += 1
+    
+    last_point = ordering[-1]
+    if (core_distances_plot[last_point] * 1.5
```

Comment:
Yes, it's a magic number which I've used in two places, and I'm not comfortable with it, i.e. I have no basis for it. Will need to do more tests to find a better _empirically_ sane value. It can also be an input parameter (which most probably will always be left as default).